napari/_qt/qt_preferences.py

```diff
@@ -0,0 +1,54 @@
+"""Preferences dialog.
+"""
+
+from qtpy.QtCore import Signal
+from qtpy.QtWidgets import (
+    QDialog,
+    QHBoxLayout,
+    QListWidget,
+    QPushButton,
+    QStackedWidget,
+    QVBoxLayout,
+)
+
+
+class PreferencesDialog(QDialog):
```

Comment:
is this stub actually going to stay in this PR?  or is it going to be removed before merge?

Reply:
Removing.